docs/scos/dev/migration-concepts/migrate-to-paas/step-13-migrate-database.md

```diff
@@ -0,0 +1,104 @@
+---
+title: 'Step 13: Migrate database'
+description: To migrate to PaaS, one of the steps, is migrating the database
+template: howto-guide-template
+---
+
+After you have [chosen a multi-store setup](/docs/scos/dev/migration-concepts/migrate-to-paas/step-12-choose-a-multi-store-setup.html), your next step is migrating the database. As Spryker PaaS supports only MariaDB, if you have MySQL or PostgreSQL, you must migrate to MariaDB. 
+
+## Migrating MySQL or MariDB to MariaDB
+
+To migrate the MySQL or MariaDB database to MariaDB in PaaS, follow these steps.
+
+1. Make sure the version of MySQL or MariaDB in the old system is compatible with the new Spryker Cloud system.
+2. Run the following queries against the source DB and save this data. You'll need them later to validate that the restoration went successfully:
+
+```php
+<?php
+$dbName = 'database_maria';
+$db = new PDO("mysql:host=localhost;dbname=$dbName", 'root', 'password');
+$tableNames = $db->query("SELECT TABLE_NAME FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_SCHEMA = '" . $dbName . "'")->fetchAll(PDO::FETCH_ASSOC);
+$tablesCount = count($tableNames);
+foreach ($tableNames as $key => $tableName){
+	$count = $db->query("SELECT count(*) as `count` from " . $tableName['TABLE_NAME'])->fetchColumn();
+    file_put_contents($dbName . "-health-check.csv", $tableName['TABLE_NAME'] . "," . $count . PHP_EOL, FILE_APPEND);
+    echo $tableName['TABLE_NAME'] . "," . $count . " - " . $key . "/" . $tablesCount . PHP_EOL;
+}
+
+```
+3. Create a database snapshot like in this example:
+
+```bash
+mysqldump -u local_user \
+    --databases database_name \
+    --single-transaction \
+    --compress \
+    --order-by-primary  \
+    --column-statistics=0 \
+    -plocal_password | gzip > /path/to/outputfile.sql.gz
+```
+For additional information about this step, see [Importing data from a MariaDB or MySQL database to a MariaDB or MySQL DB instance](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/MySQL.Procedural.Importing.SmallExisting.html).
+4. Upload the database dump to an AWS S3 bucket of the production environment using the AWS Management Console.
+5. Connect to the EC2 instance. For details on how to do this, see [Create an Amazon EC2 instance and copy the compressed database](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/MySQL.Procedural.Importing.NonRDSRepl.html#MySQL.Procedural.Importing.Import.Database).
+
+Optional: For the Spryker Cloud environment, we recommend using a Scheduler instance for backup/restore purposes.
+* AWS → EC2 → Instances → <env_name>-<[/d/w]*> → Connect via Session Manager
+  ![Scheduler Container example](https://spryker.s3.eu-central-1.amazonaws.com/docs/scos/dev/migration-concepts/migrate-to-paas/scheduler-container-example.png)
+* Find AWS credentials here:
+  ![AWS S3 credentials](https://spryker.s3.eu-central-1.amazonaws.com/docs/scos/dev/migration-concepts/migrate-to-paas/aws-s3-credentials.png)
```

Comment:
What do these steps mean? Can you elaborate more on each of them (what exactly is to be done and why)? As if you use just menu items, it is not very clear what exactly should be done and why